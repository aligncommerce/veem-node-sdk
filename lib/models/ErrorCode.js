import ErrorCodeResponse from './ErrorCodeResponse'

class ErrorCode {}

ErrorCode.response = ErrorCodeResponse

export default ErrorCode
