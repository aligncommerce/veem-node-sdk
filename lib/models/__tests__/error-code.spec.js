import ErrorCode from 'models/error-code'

describe('ErrorCode Model', () => {
  describe('response', () => {
    it('should match the snapshot', () => {
      expect(ErrorCode.response.schema).toMatchSnapshot()
    })

    it('should define a valid schema', () => {
      expect(ErrorCode.response.schema).to.be.validJsonSchema
    })
  })
})
