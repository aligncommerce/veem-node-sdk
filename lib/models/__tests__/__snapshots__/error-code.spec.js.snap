// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorCode Model response should match the snapshot 1`] = `
Object {
  "properties": Object {
    "errorCode": Object {
      "type": "number",
    },
    "errorMessage": Object {
      "type": "string",
    },
  },
  "title": "ErrorCodeResponse",
  "type": "object",
}
`;
